MekHQ/src/mekhq/campaign/mission/ScenarioForceTemplate.java

```diff
@@ -598,6 +598,8 @@ public static ScenarioForceTemplate getDefaultReinforcementsTemplate() {
         rft.setForceName(REINFORCEMENT_TEMPLATE_ID);
         rft.setGenerationMethod(ForceGenerationMethod.PlayerSupplied.ordinal());
         rft.setGenerationOrder(1);
+        rft.setActualDeploymentZone(Board.START_NONE);
+        rft.setSyncDeploymentType(SynchronizedDeploymentType.None);
```

Comment:
Wouldn't this now generate a random edge to enter from? Do we want it to instead be based on the player's edge as the base?

Reply:
That's the idea, yes. This is the default template. If someone wants to define a scenario template where the reinforcements come from a fixed position or synchronized with the primary force, they can do so by creating a "Reinforcements" force template within the scenario template, which will override this default template.